src/document.h

```diff
@@ -1107,7 +1107,11 @@ struct Document {
                 ApplyEditFilter();
                 return nullptr;
             case A_FILTERS: SetSearchFilter(true); return nullptr;
-            case A_FILTEROFF: SetSearchFilter(false); return nullptr;
+            case A_FILTEROFF: 
+                loopallcells(c) c->text.filtered = false;

```

Comment:
This appears to just copy code from the body of that function without any change in functionality? Can you tell me what this is supposed to do? And if you want to change something, better to do so inside the function.

Reply:
This is right. I just thought that is should look more generic because the function suggests that it just deals with the search filter not other filters (e.g. by cell color). But the function remains the same. So I will close this PR for now.

Reply:
sorry, still don't understand the intention.. it makes the code a lot less maintainable by having 2 copies of the same.. its less generic in an abstraction sense..

Reply:
Better I had had second thoughts about it...